plugins/hls-cabal-fmt-plugin/hls-cabal-fmt-plugin.cabal

```diff
@@ -0,0 +1,44 @@
+cabal-version:      2.4
+name:               hls-cabal-fmt-plugin
+version:            0.1.0.0
+synopsis:           TODO
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:             The Haskell IDE Team
+copyright:          The Haskell IDE Team
+maintainer:         jana.chadt@nets.at
+category:           Development
+build-type:         Simple
+extra-source-files:
+  LICENSE
+
+library
+  exposed-modules:  Ide.Plugin.CabalFmt
+  hs-source-dirs:   src
+  build-depends:
+    , base            >=4.12     && <5
+    , filepath
+    , ghcide           >=1.2 && <1.5
+    , hls-plugin-api  ^>=1.1
+    , lsp-types
+    , text
+    , transformers
+    , process
+    , cabal-fmt
+
+  default-language: Haskell2010
```

Comment:
BTW, it seems like this file itself isn't formatted by cabal-fmt.

Reply:
Thanks!